eng/native/configureplatform.cmake

```
@@ -413,6 +420,8 @@ if(CLR_CMAKE_TARGET_UNIX)
         set(CLR_CMAKE_TARGET_UNIX_X86 1)
     elseif(CLR_CMAKE_TARGET_ARCH STREQUAL s390x)
         set(CLR_CMAKE_TARGET_UNIX_S390X 1)
+    elseif(CLR_CMAKE_TARGET_ARCH STREQUAL ppc64le)
+	set(CLR_CMAKE_TARGET_UNIX_POWERPC64 1)
```

spaces for indentation:
```suggestion
        set(CLR_CMAKE_TARGET_UNIX_POWERPC64 1)
```